.gitignore

```diff
@@ -26,6 +26,8 @@
 # Jetbrains doesn't say to ignore these, but it seems like they shouldn't be committed
 **/.idea/**/watcherTasks.xml
 
+.vscode
```

Comment:
I don't know if it makes sense to ignore the whole directory.
There are a couple of things that are useful to share between people in the team (e.g. build tasks, extensions, format-on-save etc.)

Why don't you just commit your project specific settings?

Reply:
I typically ignore the root `.vscode` folder until someone has those reusable config items mentioned. In my case I'm using VsCodium which needs to leverage an environment path-specific debugger. Folks don't need my dev environment items in their codebase.

If a useful item pops up from another dev, they could add a caveat line to allow it.